doc/source/conf.py

```diff
@@ -83,7 +88,6 @@
     "sphinx.ext.graphviz",
     "sphinx_reredirects",
     "ansys_sphinx_theme.extension.linkcode",
-    "sphinx_design",
```

Comment:
Why are you deleting this? Is it because it is already loaded by ``ansys-sphinx-theme``?

Reply:
It's because it appeared twice in the extensions.

Reply:
I didn't realise 😅